addons/portal/views/portal_templates.xml

```diff
@@ -225,27 +234,100 @@
             <div class="o_portal_my_home">
                 <div class="oe_structure" id="oe_structure_portal_my_home_1"/>
                 <div class="o_portal_docs row g-2">
-                    <div t-if="portal_alert_category_enable" class="o_portal_category row g-2 mt-3" id="portal_alert_category"/>
                     <div t-if="portal_client_category_enable" class="o_portal_category row g-2 mt-3" id="portal_client_category"/>
-                    <div t-if="portal_service_category_enable" class="o_portal_category row g-2 mt-3" id="portal_service_category"/>
-                    <div t-if="portal_vendor_category_enable" class="o_portal_category row g-2 mt-3" id="portal_vendor_category"/>
-                    <div class="o_portal_category row g-2 mt-3" id="portal_common_category">
-                        <t t-call="portal.portal_docs_entry" t-if="False"/>
-                        <t t-call="portal.portal_docs_entry">
-                            <t t-set="icon" t-value="'/portal/static/src/img/portal-connection.svg'"/>
-                            <t t-set="title">Connection &amp; Security</t>
-                            <t t-set="text">Configure your connection parameters</t>
-                            <t t-set="url" t-value="'/my/security'"/>
-                            <t t-set="config_card" t-value="True"/>
-                        </t>
+                    <div class="row row-cols-1 g-3">
+                        <!-- Alert Section -->
+                        <div class="col-12">
+                            <div class="row row-cols-1 g-3">
+                                <div t-foreach="portal_entries" t-as="entry" t-if="entry.isAlert" class="col">
+                                    <div t-attf-class="o_portal_index_card d-flex gap-2 gap-md-3 py-3 pe-2 px-md-3 h-100 rounded text-decoration-none #{entry.bg_color if entry.bg_color else 'bg-100'}">
+                                        <!-- Column for the Icon -->
+                                        <div class="col-auto d-flex align-items-center">
+                                            <div t-if="entry.img" class="o_portal_icon">
+                                                <span t-field="entry.img"
+                                                      t-options="{'widget': 'image', 'width': 40, 'height': 40}"
+                                                      role="img"
+                                                      style="display: block; width: 40px; height: auto;"
+                                                      t-att-aria-label="'Icon for %s' % entry.title"
```

Comment:
Careful about translations. Doing a `t-att-` won't magically make this translatable as this can contain any custom code. Use a `t-attf-`, those are translatable, and it's more readable :+1: